### ISSUE TITLE
Como traduzir "the programmer"?

### ISSUE THREAD

**Issue by hsribei** [rust-br/rust-book-pt-br#57]
Me permiti o uso de "a programadora" na instância que encontrei do termo, seguindo o costume anglófono moderno de `she`/`her` para o caso neutro.

**Comment by leoyvens** [comment]
Vejo mais o uso the they/their mesmo que no singular como pronome neutro, em português não sei o que seria o ideal, talvez tentar reformular a frase para evitar o uso do pronome? Do tipo "quem está programando".

**Comment by darakeon** [comment]
Já amo vocês...

**Comment by miguelzinhe** [comment]
CARAMBAAAAA! ❤️

**Comment by medeirosjoaquim** [comment]
Concordo com @leodasvacas.  Pelo que vi, em alguns usos poderia ser usado "quem programa" ou até mesmo omitir a pessoa (essa forma parece encaixar melhor).

ex: 
ch17-01-what-is-oo.md

> This enables the programmer to change and refactor an object’s internals without needing ...
> Isso permite mudar e refatorar um objeto ...

ch15-03-drop.md

> In some languages, the programmer must call code to free memory or resources every time they ...
> Em algumas linguagens, é preciso invocar código para liberar memória ou recursos toda vez ... 

ch14-01-release-profiles.md
> predefined and customizable profiles with different configurations that allow a programmer to have more control over various...
> perfis predefinidos e customizáveis com diferentes configurações que permitem quem programa ter mais controle sobre vários...
